### PR TITLE
Uncommented code needed to get Moves

### LINKED ISSUE #59: Display Pokemon moves in Tab 4
**Is your feature request related to a problem? Please describe.**
Moves panel does not have fetched data yet.

**Describe the solution you'd like**
Fetch pokemon moves and display info in tab.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Uncommented function definition.
2. Made type data available for moves tab.

## Purpose
Display type of pokemon's moves.

## Approach
Destructure `type` out of move data.

Closes #59

### CODE REVIEW

File: src/components/PokeApp.js [hunk 1]
```diff
@@ -66,8 +68,39 @@ function PokeApp() {
...
+      const moves = poke.moves;
+      // get 4 moves from all moves a pokemon can learn
+      const fourMoves = getPokemonMoveset(moves, 4);
+      return fourMoves;
+    })
+  }
+
   useEffect(() => {
     // get all data: pagination/scroll, pokemon urls array
+    getMovesByPokemon(1);
```
Comment (dannysanchez559) [inline src/components/PokeApp.js:103]: Is this line hardcoding to show pokemon dexNo 1 each time?

**Review by dannysanchez559 — APPROVED** [review]

File: src/components/PokeApp.js [hunk 2]
```diff
@@ -66,8 +68,39 @@ function PokeApp() {
...
+    // return a moveSet array
+    return moveSet;
+  }
+
+  const getMovesByPokemon = async (dexNo) => {
+    const moves = allPokemons.map((poke,i)=>{
+      const moves = poke.moves;
+      // get 4 moves from all moves a pokemon can learn
+      const fourMoves = getPokemonMoveset(moves, 4);
+      return fourMoves;
```
Comment (Gjhnsn) [inline src/components/PokeApp.js:97]: Good way to get the 4 moves needed

**Review by Gjhnsn — APPROVED** [review]

File: src/components/Moves.js [hunk 3]
```diff
@@ -12,12 +12,13 @@ const Moves = ({ moveSet }) => {
   };
 
   const moveInfo = moveSet.map((moveObj) => {
-    const { accuracy, name, power, pp, flavor_text_entries } = moveObj;
+    const { accuracy, name, power, pp, flavor_text_entries, type } = moveObj;
     // call function to loop through flavor_text_entries and find "en"
     return (
-      <ul>
+      <ul key={name}>
```
Comment (BlackCubes) [inline src/components/Moves.js:18]: PogU

File: src/components/PokeApp.js [hunk 4]
```diff
@@ -66,6 +67,35 @@ function PokeApp() {
...
+
+  const getMovesByPokemon = async (dexNo) => {
+    const moves = allPokemons.map((poke, i) => {
+      const moves = poke.moves;
+      // get 4 moves from all moves a pokemon can learn
+      const fourMoves = getPokemonMoveset(moves, 4);
+      return fourMoves;
+    });
+  };
+
```
Comment (BlackCubes) [inline src/components/PokeApp.js:98]: 7346-pepe-hmmm

File: src/styles/_typography.scss [hunk 5]
```diff
@@ -1,7 +1,5 @@
 @import url("https://fonts.googleapis.com/css2?family=Nunito:wght@600;800&family=Raleway:wght@500;700&family=Rubik&display=swap");
 
-// 1 rem = 16px
-
```
Comment (BlackCubes) [inline src/styles/_typography.scss:4]: Good deletion!

**Review by BlackCubes — COMMENTED** [review]
Curious about the n-moves API. Good job!
pause-champ

**Pull request merged**